proto/regen/ecocredit/basket/v1/tx.proto

```diff
@@ -155,3 +161,23 @@ message MsgTakeResponse {
   // credits are the credits taken out of the basket.
   repeated BasketCredit credits = 1;
 }
+
+// MsgUpdateBasketFee is the Msg/UpdateBasketFee request type.
+// Since Revision 1
+message MsgUpdateBasketFee {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address of the governance account.
+  string authority = 1;
+
+  // basket_fee defines a list of coin fees that can be used in the basket
+  // creation fee.
+  repeated cosmos.base.v1beta1.Coin basket_fee = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+}
+
+// MsgUpdateBasketFeeResponse is the Msg/UpdateBasketFee response type.
+// Since Revision 1
+message MsgUpdateBasketFeeResponse {}
```

Comment:
```suggestion
// MsgUpdateBasketFees is the Msg/UpdateBasketFees request type.
// Since Revision 1
message MsgUpdateBasketFees {
  option (cosmos.msg.v1.signer) = "authority";

  // authority is the address of the governance account.
  string authority = 1;

  // basket_fees defines a list of coin fees that can be used in the basket
  // creation fee.
  repeated cosmos.base.v1beta1.Coin basket_fees = 2 [
    (gogoproto.nullable) = false,
    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
  ];
}

// MsgUpdateBasketFeesResponse is the Msg/UpdateBasketFees response type.
// Since Revision 1
message MsgUpdateBasketFeesResponse {}
```